ui/src/notifications/rules/actions/thunks.ts

```diff
@@ -45,7 +46,9 @@ import {
   RuleEntities,
   ResourceType,
 } from 'src/types'
-import {incrementCloneName} from 'src/utils/naming'
+
+// Constants
+import {LIMIT} from 'src/resources/constants'
```

Comment:
i know this is how we normally do this, but after watching russ bring down production, i dont think this limit is shared across all  api endpoints. some are 100, some are 200, which means we have no guarantee. Might be better to add a new const like RULE_LIMIT

Reply:
yes i agree.

Reply:
Took a look at resource maxes in the golang.  You're right.  Some resources are `500` some are `100`